Documentation/how-to-build-and-run-ilcompiler-in-console-shell-prompt.md

```diff
@@ -53,3 +54,18 @@ From the shell/command prompt, issue the following commands to generate the nati
     dotnet restore
     dotnet compile --native --cpp --ilcpath bin\Product\Windows_NT.x64.Debug\.nuget\publish1
 ```
+
+## Workarounds for linker errors on Windows ##
+
+If you are seeing errors such as: 
+
+```
+    LINK : fatal error LNK1104: cannot open file 'kernel32.lib'
+    Linking of intermediate files failed.
+```
+
+There are a few workarounds you might try:
+ - Make sure you run these commands from a VS developer X64 command prompt instead of a vanilla command prompt
+ - Search for the missing lib files in your SDK, for example under C:\Program Files (x86)\Windows Kits\10\lib. Make sure the path to these libraries is included in the LIB environment variable. It appears VS 2015 RTM developer command prompt does not correctly set the LIB paths for the 10586 Windows SDK. VS 2015 Update 1 resolves that issue, so installing it is another alternative.
```

Comment:
Once we are in "Native Tools for x64" command prompt, these errors shouldn't exist. Can you please confirm if you still see them?